pvlib/spectrum/mismatch.py

```diff
@@ -571,3 +571,128 @@ def spectral_factor_caballero(precipitable_water, airmass_absolute, aod500,
     )
     modifier = f_AM + f_AOD + f_PW  # Eq 5
     return modifier
+
+def spectral_factor_pelland(airmass_absolute, clearness_index,
+                            module_type = None, coefficients = None,
+                            min_airmass_absolute = 0.58,
+                            max_airmass_absolute = 10):
+    r"""
+    Estimate a technology-specific spectral mismatch modifier from
+    airmass and clearness index using the Pelland model, which takes the
+    following form:
+        
+        .. math::
+
+            M = a_1*kc**(a_2)*ama**(a_3),
+    
+    where M is the spectral mismatch factor, and a_1, a_2, a_3 are
+    module-specific coefficients.
+
+    The motivtion for this model is to include the effect of cloud cover on the
+    spectrum, and thus spectral mismatch. Another motivation is to develop a
+    simple parameterisation compared with existing models. Model coefficients
+    are derived using spectral irradiance and other meteorological data from
+    eight locations. These coefficients for seven modules, available here via
+    the ``module_type`` parameter, as well as more details on the model, can be
+    found in Ref. [1]_
+
+    Parameters
+    ----------
+    airmass_absolute : numeric
+        absolute (pressure-adjusted) airmass. [unitless]
```

Comment:
They use the Kasten and Young model, but I don't think that's relevant here. This part just defines variable names, and the user is free to derive values for those variables as they wish. I think, if the user wants to know specifics of any aspect of the methodology used by these authors, they could consult the paper.

Reply:
@echedey-ls actually I think I can include a brief summary of info such as this, but I'll add it into the intro above the parameters section rather than the parameters section itself